lambdas/evaluate-gpg45-scores/src/main/java/uk/gov/di/ipv/core/evaluategpg45scores/EvaluateGpg45ScoresHandler.java

```diff
@@ -259,6 +261,29 @@ private JourneyResponse checkForMatchingGpg45Profile(
         }
     }
 
+    private boolean checkRequiresAdditionalEvidence(String userId) {
+        List<VcStoreItem> vcStoreItems = userIdentityService.getVcStoreItems(userId);
+        if (!vcStoreItems.isEmpty()) {
+            String credentialIssuer =
+                    userIdentityService.getCredentialIssuerIfSingleValidEvidence(vcStoreItems);
+            if (credentialIssuer != null) {
+                boolean requiresAdditionalEvidence =
+                        configService
+                                .getCredentialIssuerActiveConnectionConfig(credentialIssuer)
+                                .isRequiresAdditionalEvidence();
+                LOGGER.info(
+                        new StringMapMessage()
+                                .with(
+                                        LOG_MESSAGE_DESCRIPTION.getFieldName(),
+                                        "This credential, which contains the only valid evidence, should not any GPG45 profile.")
```

Comment:
This message seems to be missing a word.
Also do we want to log this even if requiresAdditionalEvidence is false?

Reply:
The message has been changed to simpler and more understandable one. In case of a 'false' condition, we don't want to add log.